aspnetcore/fundamentals/logging/index.md

```diff
@@ -796,15 +796,76 @@ On Linux, this provider writes logs to */var/log/message*.
 logging.AddDebug();
 ```
 
-### EventSource provider
+### Event Source provider
 
-For apps that target ASP.NET Core 1.1.0 or later, the [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing. On Windows, it uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803). The provider is cross-platform, but there are no event collection and display tools yet for Linux or macOS.
+The [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing cross-platform. On Windows, the provider uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803).
 
 ```csharp
 logging.AddEventSourceLogger();
 ```
 
-A good way to collect and view logs is to use the [PerfView utility](https://github.com/Microsoft/perfview). There are other tools for viewing ETW logs, but PerfView provides the best experience for working with the ETW events emitted by ASP.NET Core.
+The Event Source provider is added automatically when `CreateDefaultBuilder` is called to build the host.
+
+::: moniker range=">= aspnetcore-3.0"
+
+#### dotnet trace tooling
+
+The [dotnet-trace](/dotnet/core/diagnostics/dotnet-trace) tool allows you to consume Event Source provider data.
+
+General syntax:
+
+```dotnetcli
+dotnet trace collect -p {pid} 
```

Comment:
There's a super weird escaping that's required when you pass multiple filter specs:

Here's an example:

```
dotnet trace collect -p 7804 --providers 'Microsoft-Extensions-Logging:C:3:FilterSpecs=\"Microsoft.AspNetCore.Hosting.Diagnostics:2;System.Net:2;Microsoft.EntityFrameworkCore.Database.Command:2;Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker:2;Microsoft.AspNetCore.Diagnostics\"'
```